src/main/java/no/unit/nva/publication/Environment.java

```diff
@@ -0,0 +1,22 @@
+package no.unit.nva.publication;
+
+import java.util.Optional;
+
+public class Environment {
+
+    /**
+     * Get environment variable.
+     *
+     * @param name  name of environment variable
+     * @return  optional with value of environment variable
+     */
+    public Optional<String> get(String name) {
+        String environmentVariable = System.getenv(name);
+
+        if (environmentVariable == null || environmentVariable.isEmpty()) {
```

Comment:
Always a fan of Objects::isNull :)

Reply:
Is it the case that this code is present in other projects? Should we create a utility? Possibly a Feature request more than QA

Reply:
Created a new task: https://unit.atlassian.net/browse/NP-571